python-sdk/src/astro/files/base.py

```diff
@@ -13,6 +13,7 @@
 from astro.files.locations import create_file_location
 from astro.files.locations.base import BaseFileLocation
 from astro.files.types import FileType, create_file_type
+from astro.options import LoadOptions, LoadOptionsList
```

Comment:
What is the benefit of having a class to represent a list of `LoadOptions`?

Reply:
Yeah I'm against this, it should just be a list of LoadOptions.

Reply:
Only one benefit, since now we have a list of options we need to find the relevant options to an integration class. So I have added two helper methods to access relevant classes by passing the name of the class. 
example - 
 `get_by_class_name("DeltaLoadOptions")`

Also, this is not user-facing I intend to use it within the operator.

@dimberman I'm happy to change this, but we either would require to replicate code or create a helper method in utils to get the relevant option class. But regardless we do need a method for this, where do you suggest we keep it?

Reply:
As per discussion with @tatiana and @pankajastro We need to change the implementation of LoadOptionsList to use dict instead of list to optimise the fetching of loadOption class.